utils.py

```diff
@@ -161,10 +161,12 @@ def _compare_endpoints(endpoint1, endpoint2):
     return True
 
 
-def map_dl_vlan(vlan_value):
-    """Return 4096/4096 and 0 respectively in case of any and untagged"""
+def match_field_dl_vlan(vlan_value, field_flow):
+    """Verify match in dl_vlan"""
     special = {"any": "4096/4096", "untagged": 0}
-
-    if isinstance(vlan_value, str):
-        return special.get(vlan_value, vlan_value)
-    return vlan_value
+    if vlan_value in special:
+        vlan_value = special.get(vlan_value, vlan_value)
```

Comment:
You could remove the `if` statement in the 167th line, `get` takes care of the non-existent keys in `special`.

Reply:
Ohh, right. Thank you, @Alopalao.